circleguard/visualizer.py

```diff
@@ -225,7 +227,7 @@ def paint_info(self, painter):
         Args:
            QPainter painter: The painter.
         """
-        painter.setPen(QPen(QColor(128, 128, 128), 1))
+        _pen = painter.pen()
```

Comment:
let's change all the other variables to _pen instead of pen_ as well then. _pen makes more sense